doc/build-unix.md

```diff
@@ -142,6 +142,26 @@ libqrencode (optional) can be installed with:
 
     sudo dnf install qrencode-devel
 
+Dependency Build Instructions: Arch Linux
+-------------------------------------
+Build requirements:
+
+sudo pacman -S gcc-c++ libtool make autoconf automake openssl-devel libevent-devel boost-devel libdb4-devel libdb4-cxx-devel zeromq binutils m4 make pkg-config miniupnpc colorcore
+
+opcional libs:
```

Comment:
Good point. This document is supposed to list dependencies for the install, neither of these is required, and they are not libs either.